src/patternfly/components/Splitter/splitter.scss

```diff
@@ -0,0 +1,70 @@
+.pf-c-splitter {
+  // splitter
+  --pf-c-splitter--Height: var(--pf-global--spacer--sm);
+  --pf-c-splitter--Width: 100%;
+  --pf-c-splitter--BackgroundColor: var(--pf-global--BorderColor--100);
+  --pf-c-splitter--Cursor: row-resize;
+  --pf-c-splitter--m-vertical--Height: 100%;
+  --pf-c-splitter--m-vertical--Width: var(--pf-global--spacer--sm);
+  --pf-c-splitter--m-vertical--Cursor: col-resize;
+
+  // splitter handle
+  --pf-c-splitter__handle--Top: 0;
+  --pf-c-splitter__handle--Right: 0;
+  --pf-c-splitter__handle--Bottom: 0;
+  --pf-c-splitter__handle--Left: 50%;
+  --pf-c-splitter--m-vertical__handle--Top: 50%;
+  --pf-c-splitter--m-vertical__handle--Right: 0;
+  --pf-c-splitter--m-vertical__handle--Left: 0;
+  --pf-c-splitter--m-vertical__handle--Bottom: 0;
+
+  // splitter handle line
+  --pf-c-splitter__handle-line--BackgroundColor: var(--pf-global--Color--100);
+  --pf-c-splitter__handle-line--Height: #{pf-size-prem(1px)};
+  --pf-c-splitter__handle-line--Width: #{pf-size-prem(12px)};
+  --pf-c-splitter--m-vertical__handle-line--Height: #{pf-size-prem(12px)};
+  --pf-c-splitter--m-vertical__handle-line--Width: #{pf-size-prem(1px)};
+
+  position: relative;
```

Comment:
I also wonder if the clickable area should be larger (vertically in the horizontal splitter, horizontal in the vertical splitter). Though that may be something we can define.

Reply:
Here's one way https://codepen.io/mcoker/pen/VwaKxQV

Reply:
good point, 8px is pretty small. @jessiehuff do you think we should increase the width so that there is more target area for the user ?

Reply:
> Here's one way https://codepen.io/mcoker/pen/VwaKxQV

That's real slick!